app/templates/403.html

```diff
@@ -3,4 +3,6 @@
 {% block content %}
     <h1>Sorry, you don't have access to this page.</h1>
     <p>For security reasons, you must be verifiably listed in the Yale directory as an undergraduate in order to access Yalies, CourseTable, and other YCS projects. If you are in the directory but are still seeing this message, please email <a href="mailto:yalecomputersociety@gmail.com">yalecomputersociety@gmail.com</a> for help.</p>
+    <h2>Are you an incoming Yale student, who has not yet started their first term of classes?</h2>
+    <p>Access to Yalies.io and other Y/CS products are granted after matriculation, usually in August before classes start. Please see our <a href="https://yalecomputersociety.org/newstudents">matriculation access policy</a> for more information.</p>
```

Comment:
My first thought was to make this a target=_blank link, but I think this is fine. Because from the student's perspective, there is nothing left to do on this page and the logical next step is to go to the linked FAQ page

Reply:
Spot on - moves them away from this product, to our info page, and if they want an opportunity to learn more about the parent org and other products currently offered.